client/my-sites/subscribers/components/subscriber-launchpad/subscriber-launchpad.tsx

```diff
@@ -41,4 +41,12 @@ const SubscriberLaunchpad = () => {
 	);
 };
 
+SubscriberLaunchpad.hasTranslationsAvailable = ( locale: string ) => {
+	return (
+		locale === 'en' ||
```

Comment:
A minor one: We could do something like `locale.startsWith( 'en' )` here to include more languages from the start.